prescriptor_robojudge.ipynb

```diff
@@ -3,7 +3,9 @@
   {
    "cell_type": "code",
    "execution_count": null,
```

Comment:
Ignore these "scrolled" thingies. They don't matter, and I couldn't find a way to get rid of them from within the notebook.